azure-pipelines.nightly.yml

```diff
@@ -236,6 +236,9 @@ stages:
             secret: true
           - name: VhServices:VideoApiUrl
             value: $(video_api_url)
+          - name: VhServices:ConferencePhoneNumber
+            value: $(ConferencePhoneNumber)
+            
```

Comment:
You need to add it to ALL the pipelines in both places. appSettings and acceptanceTestSettings if you are using it during the test